libs/auth/scripts/generate_dev_keys_and_certs.ts

```diff
@@ -100,6 +117,78 @@ async function generateDevVxAndVxAdminKeysAndCerts(
     signingPrivateKeyPassword: DEV_PRIVATE_KEY_PASSWORD,
   });
   await fs.writeFile(vxAdminCertAuthorityCertPath, vxAdminCertAuthorityCert);
+
+  if (forTests) {
+    // Save VxAdmin cert authority cert in DER format
+    await fs.writeFile(
+      vxAdminCertAuthorityCertPath.replace('.pem', '.der'),
+      await certPemToDer(vxAdminCertAuthorityCert)
+    );
+
+    const { electionHash } = electionFamousNames2021Fixtures.electionDefinition;
+    const users: UserWithCard[] = [
+      { role: 'system_administrator' },
+      { role: 'election_manager', electionHash },
+      { role: 'poll_worker', electionHash },
+    ];
+    for (const user of users) {
+      const prefix = user.role.replace('_', '-');
+      const cardVxPrivateKeyPath = `${outputDir}/${prefix}-card-vx-private-key.pem`;
+      const cardVxPublicKeyPath = `${outputDir}/${prefix}-card-vx-public-key.der`;
+      const cardVxCertPath = `${outputDir}/${prefix}-card-vx-cert.der`;
+      const cardVxAdminPrivateKeyPath = `${outputDir}/${prefix}-card-vx-admin-private-key.pem`;
+      const cardVxAdminPublicKeyPath = `${outputDir}/${prefix}-card-vx-admin-public-key.der`;
+      const cardVxAdminCertPath = `${outputDir}/${prefix}-card-vx-admin-cert.der`;
```

Comment:
At this point in my review, I am a little confused why we are generating all the `cardVxAdmin` keys as fixtures rather than generating them as part of the tests, but I assume my review will reveal this.